hadoop-ozone/ozone-recon/src/main/java/org/apache/hadoop/ozone/recon/ReconServerConfigKeys.java

```diff
@@ -114,7 +114,7 @@
 
   public static final String OZONE_RECON_TASK_THREAD_COUNT_KEY =
       "ozone.recon.task.thread.count";
-  public static final int OZONE_RECON_TASK_THREAD_COUNT_DEFAULT = 1;
+  public static final int OZONE_RECON_TASK_THREAD_COUNT_DEFAULT = 5;
```

Comment:
This should be set to a number of cores multiple instead.

Reply:
Sure, I can address this later when the number of tasks becomes sizable. Right now, we have only 2.